tensorflow/contrib/verbs/rdma.cc

```diff
@@ -471,8 +477,12 @@ void RdmaChannel::Connect(const RdmaAddress& remoteAddr) {
     attr.dest_qp_num = remoteAddr.qpn;
     attr.rq_psn = remoteAddr.psn;
     attr.max_dest_rd_atomic = 1;
-    attr.min_rnr_timer = 12;
-    attr.ah_attr.is_global = 0;
+    attr.min_rnr_timer = 12;   
+    attr.ah_attr.is_global = 1;
+    attr.ah_attr.grh.dgid.global.subnet_prefix = remoteAddr.snp;
+    attr.ah_attr.grh.dgid.global.interface_id = remoteAddr.iid;
+    attr.ah_attr.grh.flow_label = 0;
+    attr.ah_attr.grh.hop_limit = 3; 
     attr.ah_attr.dlid = remoteAddr.lid;
```

Comment:
Thanks @il-ch, sorry I was mistaken (not related to IB as the fabric is UDP-IP-ETH).
So I recommend we set the highest number from hop_limit (0xFF).
I think this the case when you set hop_limit=1 in RoCE (I think this code is in the verbs).